test/test_buffer.cpp

```diff
@@ -163,9 +168,9 @@ BOOST_AUTO_TEST_CASE(create_subbuffer)
 
 BOOST_AUTO_TEST_CASE(create_buffer_doctest)
 {
-//! [constructor]
-boost::compute::buffer buf(context, 32 * sizeof(float));
-//! [constructor]
+    //! [constructor]
+    boost::compute::buffer buf(context, 32 * sizeof(float));
+    //! [constructor]
```

Comment:
Any reason for this change? This will mess up the formatting when used as a source-code snippet in the generated documentation.